Common/Securities/Option/OptionStrategyPositionGroupBuyingPowerModel.cs

```diff
@@ -205,15 +218,27 @@ public override InitialMargin GetInitialMarginRequirement(PositionGroupInitialMa
                 return new InitialMargin(margin.Value);
             }
             else if (_optionStrategy.Name == OptionStrategyDefinitions.BearCallSpread.Name
-                || _optionStrategy.Name == OptionStrategyDefinitions.BullCallSpread.Name
-                || _optionStrategy.Name == OptionStrategyDefinitions.CallCalendarSpread.Name)
+                || _optionStrategy.Name == OptionStrategyDefinitions.BullCallSpread.Name)
             {
                 var result = GetLongCallShortCallStrikeDifferenceMargin(parameters.PositionGroup, parameters.Portfolio);
                 return new InitialMargin(result);
             }
-            else if (_optionStrategy.Name == OptionStrategyDefinitions.BearPutSpread.Name
-                || _optionStrategy.Name == OptionStrategyDefinitions.BullPutSpread.Name
+            else if (_optionStrategy.Name == OptionStrategyDefinitions.CallCalendarSpread.Name
                 || _optionStrategy.Name == OptionStrategyDefinitions.PutCalendarSpread.Name)
+            {
+                return new InitialMargin(0);
+            }
+            else if (_optionStrategy.Name == OptionStrategyDefinitions.ShortCallCalendarSpread.Name
+                || _optionStrategy.Name == OptionStrategyDefinitions.ShortPutCalendarSpread.Name)
+            {
+                var shortCall = parameters.PositionGroup.Positions.Single(position => position.Quantity < 0);
+                var shortCallSecurity = (Option)parameters.Portfolio.Securities[shortCall.Symbol];
```

Comment:
`shortCall` minor but this should be `shortOptionContract` given it's shared with the `put` version, same for the maintenance margin